source/includes/monitoring_JMX.kt

```diff
@@ -0,0 +1,30 @@
+import com.mongodb.kotlin.client.MongoClient
+import org.bson.json.JsonWriterSettings
+import com.mongodb.event.*
+import com.mongodb.MongoClientSettings
+import com.mongodb.ConnectionString
+import com.mongodb.management.JMXConnectionPoolListener
+
+
+fun main() {
+    val uri = "<connection string uri>"
+
+// Instantiate your JMX listener
+    val connectionPoolListener = JMXConnectionPoolListener()
+
+// Include the listener in your client settings
+    val settings = MongoClientSettings.builder()
+        .applyConnectionString(ConnectionString(uri))
+        .addCommandListener(connectionPoolListener)
+        .build()
+
```

Comment:
I think you're missing the part where you use the settings to build a MongoClient and use the driver (like you did in https://github.com/mongodb/docs-kotlin-sync/pull/65/files#diff-d051e6075a2c3417d2b256e95730eee1b2ef581306282f4469181eb5a946c0b8R35-R42)